R-packages/evalcast/R/plot_calibration.R

```diff
@@ -100,13 +102,21 @@ plot_coverage <- function(scorecards, alpha = 0.2, type = c("all", "one")) {
       geom_abline(slope = 1, intercept = 0) +
       geom_vline(xintercept = 1 - alpha, lty = 2) +
       facet_wrap(~ forecast_date) +
-      theme_bw() +
       xlim(0, 1) +
       ylim(0, 1) +
-      labs(x = "Nominal coverage", y = "Coverage proportion")
+      labs(x = "Nominal coverage", y = "Empirical coverage") +
+      theme_bw() + theme(legend.position = legend.position)
```

Comment:
should add theme to line with @importFrom ggplot2 so that R CMD check doesn't complain

Reply:
Ok sure.